src/log.js

```diff
@@ -1,4 +1,5 @@
+let adb = require('./adb');
+
 module.exports = function log(req, res) {
-  // TODO
-  res.status(501).send('501 Not Implemented');
+  adb.logcat().pipe(res);
```

Comment:
That's a good point. It'll happen the other way around (if logcat stream ever ends then request will end), but we need some special handling for if someone closes the request.